### ISSUE TITLE
FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated

### ISSUE THREAD

**Issue by giuseppe0525** [dieterich-lab/DCC#57]
Hi,

I got a warning message when I was running DCC and I want to know if the problem would affect the result. The warning is:
> /home/admin_g/.local/lib/python2.7/site-packages/DCC-0.4.6-py2.7.egg/DCC/circFilter.py:80: FutureWarning: Using a non-tuple sequence for multidimensional indexing is deprecated; use `arr[tuple(seq)]` instead of `arr[seq]`. In the future this will be interpreted as an array index, `arr[np.array(seq)]`, which will result either in an error or a different result.

In addition, there're many messages in the log file saying 

> circRNA start position does not have mapped read counts, treated as 0.

I don't know whether these two messages are relevant or not.

I run the command similar to the default:

> DCC @samplesheet -mt1 @mate1 -mt2 @mate2 -R [Repeats].gtf  -an [Annotation].gtf 
> -Pi -F -M -Nr 1 1 -fg -G -A [Reference].fa 

By the way, there's a minor issue when installing DCC if pysam is not installed before. The setup program will download the latest version of pysam, which is 0.15.0, and the installation will be terminated because DCC can not be built under pysam-0.15.0.

Regards,
Giuseppe

**Comment by tjakobi** [comment]
Dear @giuseppe0525,

thank you for reporting the warning. In essence, DCC is using a `numpy`-package routine that has been marked as deprecated and as of now just produces this warning with no influence on the results. However, this is a piece of code I will have to update in time to make it compatible with the new `numpy` version.

The start position "read count" message can safely be ignored, I should maybe silence that one at some point and introduce a verbose mode.

Regarding the pysam error, that's somewhat tricky and not directly related to DCC. The `HTSeq` packages requires an older version of `pysam`. This error has been fixed in the [circtools](https://github.com/dieterich-lab/circtools) package but I did not have the time yet to port the fix over to the bare DCC installation.

Thank you,
Tobias
